src/main/java/io/anserini/docgen/DocGen.java

```diff
@@ -0,0 +1,59 @@
+/**
```

Comment:
How about `io.anserini.doc` package?
I would prefer "verbs" for classes with `main` that we run - like `GenerateRegressionDocs`
Should this go in `src/main/java/` or  `src/test/java/`?
When we distribute Anserini artifact (e.g., on Maven central), I don't think this needs to be part of it?